settings.gradle.kts

```diff
@@ -17,7 +17,7 @@ dependencyResolutionManagement {
 
 rootProject.name = "sentry-root"
 rootProject.buildFileName = "build.gradle.kts"
-
+includeBuild("build-logic")
```

Comment:
Is it common convention to use dashes here? I know we have `buildSrc`(camel case) and I'm wondering if we should align the naming here.